package.json

```diff
@@ -2,6 +2,8 @@
   "name": "@finos/git-proxy",
   "version": "1.11.0",
   "description": "Deploy custom push protections and policies on top of Git.",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
```

Comment:
moving to `dist` would break existing usecases
not everything is exported or accessible via main/index atm

Reply:
I'd love to go with this for a cut of v2 some time in the future